src/components/NavBar.js

```diff
@@ -3,19 +3,18 @@ import { NavLink } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 const NavBar = ({ loggedIn }) => (
-    <div>
-      {!loggedIn ? (
-          <nav>
-            <NavLink to="/register">Register</NavLink>
-            <NavLink to="/login">Login</NavLink>
-          </nav>
-      ) : (
-          <nav>
-            <NavLink to="/logout">Logout</NavLink>
-          </nav>
-      )}
-
-    </div>
+  <div>
+    {!loggedIn ? (
+      <nav>
+        <NavLink to="/register">Register</NavLink>
+        <NavLink to="/login">Login</NavLink>
+      </nav>
+    ) : (
+      <nav>
+        <NavLink to="/logout">Logout</NavLink>
+      </nav>
+    )}
```

Comment:
there might be some conflicts/replacements of this after @sampler36 PR merge but not an issue

Reply:
yeah, was expecting those. will resolve once it's my turn to merge 👍